pkgs/development/python-modules/binwalk/default.nix

```diff
@@ -22,13 +22,13 @@
 
 buildPythonPackage rec {
   pname = "binwalk";
-  version = "2.3.2";
+  version = "27";
```

Comment:
Version seems incorrect: should probably be [`2.3.3`](https://github.com/ReFirmLabs/binwalk/releases/tag/v2.3.3) instead.

Reply:
Converted this to draft. It looks like something is wrong with the default updateScript, @FRidh @jonringer .

I get the same changes if I run:

```
nix-shell maintainers/scripts/update.nix --argstr path python38Packages.binwalk
```

Reply:
it's because they did a release: https://github.com/ReFirmLabs/binwalk/releases/tag/python27,
and the logic is assuming that python is just an arbitrary prefix, and 27 is the major version number

I'm not really sure what about this other than start having a denylist of (owner, repo, tag).

Reply:
cc @FRidh how do you feel about adding a `--target latest` choice to the update script (and have it be the default). For the "_get_latest_version_github" logic, we can just use the date of release, and I think pypi has something similar.

Pypi has `upload_time`: https://pypi.org/pypi/flake8/json

Reply:
Don't use the date please, sometimes a release is newer but not the latest because it is unstable. You can use this: https://docs.github.com/en/rest/reference/repos#get-the-latest-release